lnbits/core/templates/core/index.html

```diff
@@ -33,6 +34,7 @@
         {% endif %}
```

Comment:
- maybe show an warning that new user creation is not allowed on this instance
- the read-only landing page can create confusion

![image](https://github.com/lnbits/lnbits/assets/2951406/5c5cef64-9652-408d-b1af-4a7e544b9843)

Reply:
I prefer to keep the design clean on the front page and I feel it is nicer not to put any warning there.

Reply:
OK, with the Login PR https://github.com/lnbits/lnbits/pull/1653  there is going to be a Login form (even if new users are not allowed)